fineract-provider/src/main/java/org/apache/fineract/infrastructure/campaigns/email/ScheduledEmailConstants.java

```diff
@@ -79,8 +79,10 @@
             PREVIOUS_RUN_ERROR_LOG, PREVIOUS_RUN_ERROR_MESSAGE, NUMBER_OF_RUNS)));
 
     // report mailing job configuration names
-    public static final String GMAIL_SMTP_SERVER = "GMAIL_SMTP_SERVER";
```

Comment:
Why rename this? Am curious, because I don't know anything about this functionality.. do you happen to know if there is any documentation about how this works? I would love to understand this better to be able to set up outgoing email on https://www.Fineract.dev.. ;)

Is this because there's actually nothing Gmail specific, and this can be any SMTP server? Or is there both Gmail server specific and general SMTP support in Fineract?

Reply:
Same that above.

Unfortunately I don't have documentation, but i pass the past couple of day reading and understanding the functionality and found some in the Wiki:

https://mifosforge.jira.com/wiki/spaces/docs/pages/67895356/Manage+Scheduler+Jobs

![Screen Shot 2020-05-12 at 2 59 11 AM](https://user-images.githubusercontent.com/3644786/81656385-993a5500-93fc-11ea-92b5-111a1196e0c3.png)